src/app/garfile/arrival/index.njk

```diff
@@ -153,7 +153,7 @@
               </div>
               <div class="govuk-radios__item">
                 <input id="portChoiceCoordinates" name="portChoice" class="govuk-radios__input"
-                type="radio" value="No"{{ ' checked' if cookie.getGarArrivalVoyage().arrivalPort == 'ZZZZ'}}
+                type="radio" value="No"{{'checked' if cookie.getGarArrivalVoyage().arrivalPort === 'NaN NaN' and cookie.getGarArrivalVoyage().arrivalPort.length == 7 }}
```

Comment:
cookie.getGarArrivalVoyage().arrivalPort === 'NaN NaN', how the arrival port is returning NaN?